src/command_modules/azure-cli-deploymentmanager/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.9.0
```

Comment:
This should be 0.1.0 if it is the first preview release

Reply:
Other client SDKs suggest that we start from 0.9.0. 
Is there a reason there isn't consistent guidance among the various CLI teams?